.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: "Terraform"
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  terraform:
+    name: "Unit Tests"
+    runs-on: ubuntu-latest
```

Comment:
We should use specific version instead "latest"

Reply:
Feito! FOi usada imagem `ubuntu-18.04` assim como padrão criado no módulo cloudfare.